convert.sh

```
@@ -99,8 +100,23 @@ echo "creating ${fc}"
 mkdir ${dest} 2> /dev/null
 mkdir ${resources} 2> /dev/null
 
+echo "resolving manufacturer directory..."
+configTree=$(wget -qO- "https://api.github.com/repos/betaflight/config/git/trees/master?recursive=1")
+if [[ -z "$configTree" ]]; then
+    echo "failed to query betaflight/config repo tree. aborting."
+    rm -rf ${dest}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)